src/components/Form/Dropdown/Dropdown.jsx

```diff
@@ -104,15 +93,18 @@ export default class Dropdown extends ValidationElement {
   handleValidation (event, status, errorCodes) {
     let value = this.state.value
     let valid = value.length > 0 ? false : null
-    this.state.options.forEach(x => {
-      if (x.name.toLowerCase() === value.toLowerCase()) {
-        valid = true
-        value = x.name
-      }
-    })
 
-    if (valid === false) {
-      errorCodes = 'notfound'
+    if (valid !== null) {
+      this.state.options.forEach(x => {
+        if (x.name.toLowerCase() === value.toLowerCase() || x.value.toLowerCase() === value.toLowerCase()) {
```

Comment:
Yeah, I can't really figure out the meaning behind this check. What's it really checking for?